R/Report_OverviewText.R

```diff
@@ -28,6 +28,10 @@ Report_OverviewText <- function(lSetup, dfResults, lStudy) {
     group_type <- "countries"
   }
 
+  if(is_empty(amber_or_red_KRI_groups)) {
+    cat("0 of 15 groups selected. Update filter to see more data.")
```

Comment:
Is it always 15? Can possibly turn this into a programmatically generated number based on unique of groups in the respective report

Reply:
I think this number would be `length(unique(dfResults$GroupID))` because those are all the possible groups that can have flags

Reply:
instead of "selected" might say something like "0 of XX groups applicable to this filter. Update filter to see more data."